src/main/java/com/epam/emotionalhelp/controller/config/CORSConfig.java

```diff
@@ -0,0 +1,8 @@
+package com.epam.emotionalhelp.controller.config;
+
+import org.springframework.context.annotation.Configuration;
+
+@Configuration
+public class CORSConfig {
+    public static final String LOCALHOST = "http://localhost:8080";
```

Comment:
If we change `server.port`, this configuration will not work. Can we try to refactor it to `http://localhost:*` setting?